manifests/07-operator.yaml

```diff
@@ -53,3 +53,10 @@ spec:
               value: "cluster-image-registry-operator"
             - name: IMAGE
               value: docker.io/openshift/origin-docker-registry:latest
+          volumeMounts:
+            - name: proxyca
+              mountPath: /etc/pki/ca-trust/source/
```

Comment:
pretty sure the bundle you're trying to overwrite is this one:
/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem

but i guess i'm not positive which one golang is actually reading since there is also:
/usr/share/pki/ca-trust-legacy/ca-bundle.legacy.default.crt

but it should be one of those two (there are various symlinks that point to those 2 files)

Reply:
I've called this out in the proxy design doc. Will the operator container need to run `update-ca-trust extract` before it begins?

Reply:
i would not expect so, no.  pretty sure update-ca-trust extract is what produces those bundles.